modules/mod_custom/services/provider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
```

Comment:
```suggestion
<?php

/**
```
Fix PHPCS failing in drone.

Reply:
You have marked this as resolved but not really fixed it. So PHPCS still fails.